### PR TITLE
Wip alexandr

### PR DESCRIPTION
**Pull request by alex-gryadkin**
second pull request.
All the previous comments are resolved and added Database integration

### CODE REVIEW

File: application/templates/base.html [hunk 1]
```diff
@@ -2,44 +2,7 @@
...
-                  text-align: center;
-                  text-decoration: none;
-                  display: inline-block;
-                  font-size: 16px;
-                }
-                .button1 {background-color: #f44336;} /* Red */
-                .button2 {background-color: #008CBA;} /* Blue */
-                .button0 {float: right}
-            </style>
+            <link rel='stylesheet' type='text/css' href='static/style.css'>
```
Comment (kapitan-alexey) [inline application/templates/base.html]: is it proper way to connect static files to flask app?
check this:
[https://flask.palletsprojects.com/en/2.2.x/tutorial/static/](url)

**Review by kapitan-alexey — COMMENTED** [review]
I can see __pycache__ folder in PR, are you sure you're ignoring it in right way?

File: application/models.py [hunk 2]
```diff
@@ -0,0 +1,10 @@
+from application import db
+
+
+class Task(db.Model):
+    id = db.Column(db.Integer, primary_key=True)
+    task_descr = db.Column(db.String(300))
+    status = db.Column(db.String(30), index=True)
```
Comment (kapitan-alexey) [inline application/models.py]: what data will be stored in _status_ field of your _Task_ model? What are you going to have?
if it is like: **Done**, **Not Done** then mb you can have this field as **Boolean**?
or you can use this one (I would do it in this way):
[https://sqlalchemy-utils.readthedocs.io/en/latest/data_types.html#module-sqlalchemy_utils.types.choice](url)
[https://stackoverflow.com/questions/63034703/flask-and-sqlalchemys-column-type](url)
time to learn what is **Enum** in Python

**Review by kapitan-alexey — COMMENTED** [review]
Please keep your commit short and do not add new functionality during ongoing PR

File: application/models.py (hunk 2 above)
Comment (alex-gryadkin) [reply application/models.py]: done. Enum added

File: application/templates/base.html (hunk 1 above)
Comment (alex-gryadkin) [reply application/templates/base.html]: static files connection changed

**Review by kapitan-alexey — APPROVED** [review]

**Pull request merged**